pandas/core/indexing.py

```diff
@@ -2057,6 +2057,11 @@ def _convert_key(self, key, is_setter: bool = False):
         Require they keys to be the same type as the index. (so we don't
         fallback)
         """
+        # For series, unpacking key needs to result in the label.
+        # This is already the case for len(key) == 1; e.g. (1,)
```

Comment:
can you add the GH reference to this comment

For the check below, can you use `self.ndim == 1` instead of `isinstance(self.obj, ABCSeries)`.  The ndim check is about 7x faster.